internal/cli/clients/client.go

```diff
@@ -827,7 +824,6 @@ func (c *EpinioClient) Info() error {
 	c.ui.Success().
 		WithStringValue("Platform", platform).
 		WithStringValue("Kubernetes Version", kubeVersion).
-		WithStringValue("Gitea Version", giteaVersion).
```

Comment:
This requires a matching change in the testsuite (before test setup - This part still looks for this string - CI :boom: because of that)